Backpack/src/main/java/net/skyscanner/backpack/calendar2/view/CalendarDayLabelContentColor.kt

```diff
@@ -0,0 +1,45 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.calendar2.view
+
+import android.content.Context
+import android.content.res.ColorStateList
+import net.skyscanner.backpack.R
+import net.skyscanner.backpack.calendar2.CellStatus
+
+internal typealias CalendarDayLabelContentColor = (CellStatus?) -> ColorStateList
+
+internal fun CalendarDayLabelContentColor(
+  context: Context,
+): CalendarDayLabelContentColor {
+
+  val default = context.getColorStateList(R.color.bpkTextSecondary)
```

Comment:
do these need theming support?

Reply:
We keep the same approach as in original calendar and theme only those properties which were themed there. I'd removed that theming as well s it doesn't seem to be used, but I'd prefer to do these changes gradually.